Source/WebCore/cssjit/SelectorCompiler.cpp

```diff
@@ -1343,6 +1343,9 @@ static inline FunctionType addPseudoClassType(const CSSSelector& selector, Selec
                     selectorFragments = &matchesList.last();
                 }
 
+                if (subselector.matchesPseudoElement())
+                    return FunctionType::CannotCompile;
+
```

Comment:
The `if` condition line 1361 seems redundant now (it might have been invalid before). We should remove it

Reply:
Done